web/pdf_thumbnail_viewer.js

```diff
@@ -257,8 +258,8 @@ class PDFThumbnailViewer {
   }
 
   forceRendering() {
-    let visibleThumbs = this._getVisibleThumbs();
-    let thumbView = this.renderingQueue.getHighestPriority(visibleThumbs,
+    const visibleThumbs = this._getVisibleThumbs();
+    const thumbView = this.renderingQueue.getHighestPriority(visibleThumbs,
                                                            this._thumbnails,
                                                            this.scroll.down);
```

Comment:
Nit: Please fix the indentation here.

Reply:
Afterwards, please squash all commits into one as described at https://github.com/mozilla/pdf.js/wiki/Squashing-Commits so we can merge it. Thanks!